$NetBSD$

Remove unneeded include file, else it won't compile under NetBSD-1.6.x

--- appl/lib/ka-procs.c.orig	Mon Mar 14 19:33:24 2005
+++ appl/lib/ka-procs.c	Mon Mar 14 19:33:38 2005
@@ -53,7 +53,7 @@
 
 #ifdef HAVE_OPENSSL
 #include <openssl/des.h>
-#include <openssl/ui.h>
+/* #include <openssl/ui.h> */
 #include <openssl/opensslv.h>
 #endif
 
